### PR TITLE
[Bug 1157024] Fixed broken edit profile page on Mobile

### PR DESCRIPTION
**Pull request by anushbmx**
Fixes Edit profile page on Mobile which was broke due to bad styling.

I have removed the border applied to Label and made it look like other form input used in SUMO mobile version.

### CODE REVIEW

**Comment by rehandalal** [comment]
this solution doesn't work as it completely changes the styling of the form and is no longer in line with the UI that was designed for this page.

File: kitsune/sumo/static/sumo/less/mobile/users.less [hunk 1]
```diff
@@ -140,43 +140,22 @@
...
-      width: 96px;
 
       & + input,
       & + select,
       & + textarea,
       & + .value,
       & + .field-button {
-        border-radius: 0 3px 3px 0;
-        margin-left: 96px;
-        width: calc(100% - 96px);
```
Comment (rehandalal) [inline kitsune/sumo/static/sumo/less/mobile/users.less]: this should be changed to:
```css
width: calc(~"100% - 96px");
```

File: kitsune/sumo/static/sumo/less/mobile/users.less [hunk 2]
```diff
@@ -140,43 +140,22 @@
...
       &.big {
         width: 180px;
-
-        & + input,
-        & + select,
-        & + textarea,
-        & + .value,
-        & + .field-button {
-          margin-left: 150px;
-          width: calc(100% - 180px);
```
Comment (rehandalal) [inline kitsune/sumo/static/sumo/less/mobile/users.less]: likewise:
```css
width: calc(~"100% - 180px");
```

**Comment by rehandalal** [comment]
if you undo all your changes and make the two changes I mentioned that should fix the issue correctly.

the problem is less is computing those values within the calc expression.

**Comment by anushbmx** [comment]
Ok @rehandalal I had worked sometime to fix the original design but was having issues with drop downs for country and time line so I just took a look at inputs and found this simple style. Let me try again.

**Comment by anushbmx** [comment]
@rehandalal Fixed it :D it needed a little more changes. I didn't knew how to use calc() before. Thanks It was another lesson for me.

**Comment by rehandalal** [comment]
this is better, but there is still an issue. you've gotten rid of the label on the password field

File: kitsune/users/templates/users/mobile/edit_profile.html [hunk 3]
```diff
@@ -28,7 +28,6 @@
         <a class="field-button" href="{{ url('users.change_email') }}" data-next="{{ request.get_full_path() }}">{{ _('Change Email') }}</a>
       </div>
 
-      <label>{{ _('Password') }}</label>
```
Comment (rehandalal) [inline kitsune/users/templates/users/mobile/edit_profile.html]: we need to keep this

**Comment by rehandalal** [comment]
you'll need to float a bunch of things left and adjust left margins accordingly on certain elements.

**Comment by rehandalal** [comment]
While investigating I ended up fixing this locally (along with all the other `calc()`) expressions. Filed #2572. Closing this in favor of that.

**Comment by anushbmx** [comment]
ok @rehandalal .

Any idea why build failed...?

**Comment by rehandalal** [comment]
the test failure looks unrelated.
